roles/nuage_node/tasks/iptables.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: IPtables | Get iptables rules
+  command: iptables -L --wait
+  register: iptablesrules
+  always_run: yes
+
+- name: Allow traffic from overlay to underlay
+  command: /sbin/iptables -I FORWARD 1 -s {{ hostvars[groups.oo_first_master.0].openshift.master.sdn_cluster_network_cidr }} -j ACCEPT -m comment --comment "nuage-overlay-underlay" 
```

Comment:
I would suggest adding the `-w`/`--wait` flag to this and the following iptables commands. I'm wondering if it would be a good idea to flush the handlers after each command as well to avoid potential issues.

@sdodson @tbielawa this made me look back at the os_firewall module we are currently using, and we should be making use of the `--wait` flag there as well. Of course, I did not know about the `--wait` flag then :smile:  I'm also not sure if we should extend os_firewall for this use-case, or handle it within this role in an ad-hoc manner until we migrate to firewalld.